src/_data/content.json

```diff
@@ -187,6 +187,11 @@
         "image": "/assets/img/hwr_logo.png",
         "link": "https://www.hwr-berlin.de/"
       },
+      {
+        // "name": "Kunsthochschule Weißensee",
```

Comment:
// is not valid JSON. This will create a runtime error.

Reply:
hab das JSON entkommentiert und dafür den `<p>` in der default.html als Kommentar gesetzt, damit die Namen nicht unter den Logos erscheinen